jade/page-contents/navbar_content.html

```diff
@@ -641,9 +641,6 @@ <h3 class="header">Mobile Collapse Button</h3>
 
         <br>
         <h5>Initialization</h5>
-        <p>After including the sidenav-trigger line into your navbar, all you have to do now initialize the plugin. This example
```

Comment:
I do not believe this text is necessary. If it is, then we ought to include similar text in **all** of the examples.